### DISCUSSION TITLE
Backend Console: $unset field

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#339] [Help] [answered]
Ahoi!

I tried to remove a field added before (permission configuration) via the backend console. I tried it with the modifiers from documentation $unset etc. but these are adding a field "$unset". How can I do this?

```
{
  "permissions": {
    "$unset": {
      "categories": ""
    }
  }
}
```

**Comment by ericingram** [comment]
I think `$unset` has to be at the top level of the request.

Try this:

```
PUT ...
{
  "$unset": "permissions.$unset"
}
```

**Reply by josias-hosang** [reply]
I tried but got error:
`Not allowed: $unset`

**Reply by logeshswell** [reply, marked as answer]
@josias-hosang The endpoints with the prefix colon (:) doesn't allow `$unset` field. So, you can copy the values in `permissions` field and then set `"permissions": null` which clears all the allowed permissions.
Make another PUT API call to set permissions for the expected values.

**Reply by josias-hosang** [reply]
That's the way I did it. As long as this is OK, I'm fine. Thanks.
